File: Data Structures & Algorithms/anagram-groups/submission-1.java
class Solution {
    public List<List<String>> groupAnagrams(String[] strs)
    {
        Map <String, List<String>> map = new HashMap<>();
        for(String item : strs)
        {
            String sorted = Sorting(item);

            if(!map.containsKey(sorted))
            {
                map.put(sorted, new ArrayList<>());
            }

            map.get(sorted).add(item);
        }

        return new ArrayList<>(map.values());
    }
    private String Sorting(String str)
    {
        char[] chars = str.toCharArray();
        Arrays.sort(chars);
        return new String(chars);
    }
}

File: Data Structures & Algorithms/climbing-stairs/submission-1.cs
public class Solution {
    public int ClimbStairs(int n) {     
        if(n == 2)
            return  2;
        if(n == 1)
            return 1;
        if(n == 0)
            return 0;
        return (n - 1) + (n - 2);
    }
}
